dowhy/utils/plotting.py

```diff
@@ -176,6 +176,9 @@ def bar_plot(
 
 
 def _calc_arrow_width(strength: float, max_strength: float):
+    if max_strength == 0:
+        return 4.1
+
     return 0.1 + 4.0 * float(abs(strength)) / float(max_strength)
```

Comment:
Recommend a check beforehand to ensure that the `strength` is less than or equal to `max_strength`.